Core/src/Utilities/SpacePointUtility.cpp

```diff
@@ -0,0 +1,323 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Utilities/SpacePointUtility.hpp"
+
+#include <iostream>
+namespace Acts {
+
+double SpacePointUtility::differenceOfMeasurementsChecked(
```

Comment:
We usually take the `Result` object if something can fail, and check it. It could be an overkill for a double though.

Reply:
Changed it to `Result<double>`